README.md

```diff
@@ -0,0 +1,56 @@
+# Hardware interface for Dynamixels and ROS control
+
+[ROS control][] is a framework to design software control loops in [ROS][] (Robot Operating System) where the controller code is decoupled from the actual hardware.
+
+This piece of software provides a hardware interface for ROS control. it's aim is to allow generic software controllers to control a set of Dynamixel actuators.
+
+We are using it for our robots and developing it for this purpose. Version 0.0.1 is clearly no final code but it works for the basic usage.
+
+## Features
+
+- simple, ROS-style control interface for your dynamixel-based robot
+- not specific to a given number or set of actuators
+- uses radians uniformly over all Dynamixel models (no need to worry about ticks-to-angle conversion)
+- you can set an offset for each actuator's position
```

Comment:
You're right. I'm adding the example configuration in the position controller (was only in the trajectory).